podman/domain/volumes.py

```diff
@@ -34,6 +34,16 @@ def remove(self, force: Optional[bool] = None) -> None:
             APIError: when service reports an error
         """
         self.manager.remove(self.name, force=force)
+    
+    def inspect(self) -> dict:
+        """Inspect this volume
+        
```

Comment:
There are a bunch of white spaces that the linter is complaining about. could you please remove those?

Reply:
Cleaned. Currently using a bit raw editor.